Classes/Service/RecordService.php

```diff
@@ -45,7 +45,13 @@ public function get($table, $fields, $clause = null, $groupBy = null, $orderBy =
             $statement->where($clause);
         }
         if ($limit) {
-            $statement->setMaxResults($limit);
+            if (is_int($limit)) {
+                $statement->setMaxResults($limit);
+            } else {
+                $limit =  explode(',', $limit);
```

Comment:
Are you sure this is as it is intended and not something you assimilated from the EXT:news method of defining sortings and limits? The problem seems more to be that we don't have an offset argument.